nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -372,17 +379,19 @@ in {
         };
 
         user = lib.mkOption {
+          type = lib.types.nullOr lib.types.str;
```

Comment:
These additions don't have anything to do with the purpose of the PR, right? Please file them as another PR if that's correct.

Reply:
Okay, I'll move it to a another PR.